### ISSUE TITLE
The `container` property incorrectly required in Lottie component props

### ISSUE THREAD

**Issue by raimohanska** [Gamote/lottie-react#47]
In version 2.2.0, the TypeScript type `LottieOptions` inherits from `AnimationConfig` (part of lottie-web) which has a require `container` property of type `Element`. This seems like a typing bug to me.

```
Error: src/components/Button.tsx(62,12): error TS2741: Property 'container' is missing in type '{ className: string | undefined; animationData: { v: string; fr: number; ip: number; op: number; w: number; h: number; nm: string; ddd: number; assets: never[]; layers: { ddd: number; ind: number; ty: number; ... 8 more ...; bm: number; }[]; markers: never[]; }; }' but required in type 'AnimationConfig<"svg">'.
```

**Comment by Gamote** [comment]
Hey @raimohanska thank you for reporting, I will take care of it today.

**Comment by raimohanska** [comment]
Great! No hurry here and thanks for your great work. Let me know if u need help with testing or anything.

**Comment by Gamote** [comment]
Hey @raimohanska, this bug was fixed in [v2.2.1](https://github.com/Gamote/lottie-react/releases/tag/v2.2.1).
